include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -4854,6 +4854,42 @@
       "capabilities" : [ "CooperativeMatrixKHR" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpConstantCompositeReplicateEXT",
+      "class"  : "Constant-Creation",
+      "opcode" : 4461,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef",        "name" : "'Value'" }
+      ],
+      "capabilities" : [ "ReplicatedCompositesEXT" ],
+      "version" : "None"
+    },
+    {
+      "opname" : "OpSpecConstantCompositeReplicateEXT",
+      "class"  : "Constant-Creation",
+      "opcode" : 4462,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef",        "name" : "'Value'" }
+      ],
+      "capabilities" : [ "ReplicatedCompositesEXT" ],
+      "version" : "None"
+    },
+    {
+      "opname" : "OpCompositeConstructReplicateEXT",
+      "class"  : "Constant-Creation",
```

Comment:
```suggestion
      "class"  : "Composite",
```

Reply:
Done for the two "Composite" instructions.

Reply:
This looks backward. The corresponding existing instructions are:
- OpConstantComposite and OpSpecConstantComposite are in "Constant-Creation Instructions"
- OpCompositeConstruct is in "Composite Instructions".